### DISCUSSION TITLE
Translate a Placeholder

### DISCUSSION THREAD

**Discussion by caiobarnes** [GriefPrevention/GriefPrevention#2176] [General]
Hello, my friends,

I would like to know how to translate this placeholder.

Placeholder: `%griefprevention_currentclaim_ownername%`

I would like to translate the word 'Unclaimed.'

P.S.: I tried expanding GriefPrevention in the string editor, but I was not successful.

![error](https://github.com/TechFortress/GriefPrevention/assets/65726281/d7f13c87-71ef-4844-992a-cc2203b08226)

**Comment by Jikoo** [comment]
Placeholders are not provided by GP, you'll need to contact the placeholder expansion's author for support. No idea who controls it any more, original author appears to have deleted the repo it was in. The only messages GP provides would be player names, "someone(UUID)" (untranslatable) for long-offline players, and "an administrator" (translatable) for admin claim ownership.
